Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -583,6 +561,7 @@ class ToggleChronTabs: HiddenSetting {
     }
 
     override var title: NSAttributedString? {
+        // Does the below still apply?
```

Comment:
@nbhasin2 @boek Any knowledge about this?

@nishant2718 Generally, we'll try not to have random comments in our codebase. I guess it's a decent way to get the conversation started, but we should make sure that unless these will get resolved right away, we mark them as todos with context for future. Otherwise, if we go over it, we might easily forget what it's all about. :)

```suggestion
        // TODO: Check if the below is still relevant with LeanPlum having been removed.
```